contentcuration/contentcuration/tasks.py

```diff
@@ -292,24 +290,72 @@ def sendcustomemails_task(subject, message, query):
 
 
 type_mapping = {
-    "duplicate-nodes": {"task": duplicate_nodes_task, "progress_tracking": True},
-    "move-nodes": {"task": move_nodes_task, "progress_tracking": False},
-    "delete-node": {"task": delete_node_task, "progress_tracking": False},
-    "export-channel": {"task": export_channel_task, "progress_tracking": True},
-    "sync-channel": {"task": sync_channel_task, "progress_tracking": True},
-    "get-node-diff": {"task": generatenodediff_task, "progress_tracking": False},
+    "duplicate-nodes": duplicate_nodes_task,
+    "move-nodes": move_nodes_task,
+    "delete-node": delete_node_task,
+    "export-channel": export_channel_task,
+    "sync-channel": sync_channel_task,
+    "get-node-diff": generatenodediff_task,
+    "calculate-resource-size": calculate_resource_size_task,
 }
 
 if settings.RUNNING_TESTS:
     type_mapping.update(
         {
-            "test": {"task": test_task, "progress_tracking": False},
-            "error-test": {"task": error_test_task, "progress_tracking": False},
-            "progress-test": {"task": progress_test_task, "progress_tracking": True},
+            "test": test_task,
+            "error-test": error_test_task,
+            "progress-test": progress_test_task,
         }
     )
 
 
+def _build_metadata(task_args):
```

Comment:
Do we still need this anywhere?

Reply:
Do you mean the actual metadata? Now that you mention it, I think the task endpoint is using the channel ID, but we could get that off the Task model. So I'm thinking we don't

Reply:
Yeah - and I think you'd said this to me previously too.

Reply:
I've removed the `affects` data from the metadata. For now, I've kept the `args`, as that could be useful for debugging in the event of issues.